src/JsonApi/Serializer/ErrorNormalizer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\JsonApi\Serializer;
+
+use Symfony\Component\Debug\Exception\FlattenException;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class ErrorNormalizer implements NormalizerInterface
```

Comment:
You should use https://github.com/api-platform/core/blob/master/src/Problem/Serializer/ErrorNormalizerTrait.php for this class.